src/users/users.routes.ts

```diff
@@ -12,5 +16,27 @@ export function getUsersRouter(): Router {
     }
   })
 
+  userRouter.post('/login', async (req: Request, res: Response) => {
+    try{
+      const currentUser = await userService.loginUser(req.body) 
+      if(!currentUser){
+        res.status(401).json({message: 'No user with that email'})
+      } else {
```

Comment:
you could use a return statement on line 23, then there wouldnt be a need for the else block, this would make the code more readable